paasta_tools/utils.py

```diff
@@ -1694,6 +1695,12 @@ def get_taskproc(self) -> Dict:
     def get_disabled_watchers(self) -> List:
         return self.config_dict.get('disabled_watchers', [])
 
+    def get_vault_environment(self) -> str:
+        """ Get the environment name for the vault cluster
+        This must match the environment keys in the secret json files
+        used by all services in this cluster"""
+        return self.config_dict.get('vault_environment')
```

Comment:
Do you want a default here?  Otherwise I think `.get` will return `None` (breaking the `str` type of the function)

Reply:
I couldn't decide what was more dangerous tbh. I think None is probably best and I'll change it to Optional[str].